### ISSUE TITLE
`variant version` produces error

### ISSUE THREAD

**Issue by Nuru** [mumoshu/variant#63]
I installed `variant` from the release archive https://github.com/mumoshu/variant/releases/download/v0.25.0/variant_0.25.0_linux_amd64.tar.gz into a Docker container based on `alpine:3.8`.

Running `variant version` produces the error seen below:

```
$> variant version
ERRO[0000] yaml: unmarshal errors:
  line 1: cannot unmarshal !!str `0.25.0` into map[string]interface {}
yaml.Unmarshal failed: yaml: unmarshal errors:
  line 1: cannot unmarshal !!str `0.25.0` into map[string]interface {}
github.com/mumoshu/variant/pkg.ReadTaskDefFromBytes
	/go/src/github.com/mumoshu/variant/pkg/task_def_loader.go:33
github.com/mumoshu/variant/pkg.ReadTaskDefFromFile
	/go/src/github.com/mumoshu/variant/pkg/task_def_loader.go:51
github.com/mumoshu/variant/pkg/run.Dev
	/go/src/github.com/mumoshu/variant/pkg/run/run.go:82
main.main
	/go/src/github.com/mumoshu/variant/main.go:8
runtime.main
	/usr/local/go/src/runtime/proc.go:198
runtime.goexit
	/usr/local/go/src/runtime/asm_amd64.s:2361
Error while loading version
github.com/mumoshu/variant/pkg.ReadTaskDefFromFile
	/go/src/github.com/mumoshu/variant/pkg/task_def_loader.go:54
github.com/mumoshu/variant/pkg/run.Dev
	/go/src/github.com/mumoshu/variant/pkg/run/run.go:82
main.main
	/go/src/github.com/mumoshu/variant/main.go:8
runtime.main
	/usr/local/go/src/runtime/proc.go:198
runtime.goexit
	/usr/local/go/src/runtime/asm_amd64.s:2361 
panic: Error while loading version: yaml.Unmarshal failed: yaml: unmarshal errors:
  line 1: cannot unmarshal !!str `0.25.0` into map[string]interface {}: yaml: unmarshal errors:
  line 1: cannot unmarshal !!str `0.25.0` into map[string]interface {}

goroutine 1 [running]:
github.com/mumoshu/variant/pkg/run.Dev()
	/go/src/github.com/mumoshu/variant/pkg/run/run.go:86 +0x5ef
main.main()
	/go/src/github.com/mumoshu/variant/main.go:8 +0x20
```

Of course, I expected it to output something like `0.25.0`

---
I installed `variant` from the release archive https://github.com/mumoshu/variant/releases/download/v0.25.0/variant_0.25.0_darwin_amd64.tar.gz onto a Mac.

Running `variant version` produces `version`:
```
$> variant version
version
```
Of course, I expected it to output something like `0.25.0`

**Comment by mumoshu** [comment]
@Nuru Thanks for reporting! This has been fixed via https://github.com/mumoshu/variant/commit/5328823f2dfcb7a082c646b6c68ef573ef9133dd and v0.27.2

**Comment by Nuru** [comment]
@mumoshu I'm seeing this same error (without the stack trace) with Variant 0.36.4 in Alpine 3.11.5

```
 #  variant version
ERRO[0000] Error while loading version: yaml.Unmarshal failed: yaml: unmarshal errors:
  line 1: cannot unmarshal !!str `0.36.4` into map[string]interface {}: yaml: unmarshal errors:
  line 1: cannot unmarshal !!str `0.36.4` into map[string]interface {} 
```

**Comment by mumoshu** [comment]
@Nuru Hey! Thanks. but I can't reproduce it.

Are you by any chance:

- Have `VARIANTLFILE` envvar set to a file whose content is `0.36.4`
- Or have a file named `./Variantfile` whose content is `0.36.4` when the `variant version` failed?

https://github.com/mumoshu/variant/blob/139ec2bdb6ed0d8bbac2c4cb45727fffc465a539/pkg/task_def_loader.go#L48

**Comment by Nuru** [comment]
> Are you by any chance:
> 
>     * Have `VARIANTLFILE` envvar set to a file whose content is `0.36.4`
> 
>     * Or have a file named `./Variantfile` whose content is `0.36.4` when the `variant version` failed?
> 

No, but I have a file named `VERSION` in the current directory....

**Comment by mumoshu** [comment]
@Nuru Thank you so much! That revealed a nasty bug in Variant... It's fixed in v0.36.5
